src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,73 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+/** A class which encapsulates the list of
+ * tasks stored.
+ */
+public class TaskList {
+
+    /** An arraylist to store the tasks */
+    private ArrayList<Task> taskList;
+
+    /**
+     * A public constructor for TaskList which
+     * initializes the taskList to an empty Array
+     * List.
+     */
+    public TaskList() {
+        taskList = new ArrayList<>();
+    }
+
+    /**
+     * A public constructor which initializes the taskList
+     * to the given one.
+     *
+     * @param taskList The list of tasks.
+     */
+    public TaskList(ArrayList<Task> taskList) {
+
+        this.taskList = taskList;
+    }
+
+    /**
+     * Adds a task to the list of tasks.
+     *
+     * @param task The task to be added.
+     */
+    public void add(Task task) {
+        taskList.add(task);
+    }
+
+    /**
+     * Deletes a task from the list given the index of the task.
+     *
+     * @param num The index of the task to be deleted.
+     */
+    public void delete(int num) {
+
+        taskList.remove(num-1);
```

Comment:
It is better to leave some spaces between operators and characters i to improve readability.